src/routes/docs/products/functions/execute/+page.markdoc

```diff
@@ -225,11 +216,42 @@ const promise = functions.createExecution(
     );
 
 promise.then(function (response) {
-    console.log(response);
+    console.log(response); // Success
 }, function (error) {
-    console.log(error);
+    console.log(error); // Failure
 });
 ```
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/appwrite/sdk-for-go/appwrite"
+)
+
+func main() {
+	appwriteClient := appwrite.NewClient(
+		appwrite.WithProject("<PROJECT_ID>"),
+	)
+
+	appwriteFunctions := appwrite.NewFunctions(appwriteClient)
+
+	promise, err := appwriteFunctions.CreateExecution(
```

Comment:
They're not called promises in go. Use either:

1. execution
2. result
3. response